components/item/AwesomeSwiper.stories.js

```diff
@@ -1,15 +1,61 @@
 import { storiesOf } from '@storybook/vue';
-import VueI18n from 'vue-i18n';
+import Vuex from 'vuex';
 import AwesomeSwiper from './AwesomeSwiper';
+import ItemHero from './ItemHero';
 
-const i18n = new VueI18n();
+const i18n = {
+  locale: 'en',
+  messages: {
+    en: {
+      actions: {
+        close: 'close',
+        download: 'download',
+        share: 'share',
+        shareOn: 'Share on {social}'
+      }
+    }
+  }
+};
+
+const store = new Vuex.Store({
+  getters: {
+    'http/canonicalUrl': () => {
+      return 'https://www.example.org/';
+    }
+  }
+});
 
 const media = [
-  { thumbnails: { large: 'https://api.europeana.eu/api/v2/thumbnail-by-url.json?size=w400&type=IMAGE&uri=https%3A%2F%2Feuropeana1914-1918.s3.amazonaws.com%2Fattachments%2F119112%2F10265.119112.original.jpg' } },
-  { thumbnails: { large: 'https://api.europeana.eu/api/v2/thumbnail-by-url.json?size=w400&type=IMAGE&uri=https%3A%2F%2Feuropeana1914-1918.s3.amazonaws.com%2Fattachments%2F119200%2F10265.119200.original.jpg' } },
-  { thumbnails: { large: 'https://api.europeana.eu/api/v2/thumbnail-by-url.json?size=w400&type=IMAGE&uri=https%3A%2F%2Feuropeana1914-1918.s3.amazonaws.com%2Fattachments%2F119203%2F10265.119203.original.jpg' } },
-  { thumbnails: { large: 'https://api.europeana.eu/api/v2/thumbnail-by-url.json?size=w400&type=IMAGE&uri=https%3A%2F%2Feuropeana1914-1918.s3.amazonaws.com%2Fattachments%2F119639%2F10265.119639.original.jpg' } },
-  { thumbnails: { large: 'https://api.europeana.eu/api/v2/thumbnail-by-url.json?size=w400&type=IMAGE&uri=https%3A%2F%2Feuropeana1914-1918.s3.amazonaws.com%2Fattachments%2F119640%2F10265.119640.original.jpg' } }
+  {
+    about: 'https://europeana1914-1918.s3.amazonaws.com/attachments/119112/10265.119112.original.jpg',
+    thumbnails: { large: 'https://api.europeana.eu/api/v2/thumbnail-by-url.json?size=w400&type=IMAGE&uri=https%3A%2F%2Feuropeana1914-1918.s3.amazonaws.com%2Fattachments%2F119112%2F10265.119112.original.jpg' },
+    rightsStatement: 'http://creativecommons.org/licenses/by-sa/3.0/',
+    downloadUrl: 'https://proxy.europeana.eu/2020601/https___1914_1918_europeana_eu_contributions_10265?view=https%3A%2F%2Feuropeana1914-1918.s3.amazonaws.com%2Fattachments%2F119112%2F10265.119112.original.jpg&api_url=https%3A%2F%2Fapi.europeana.eu%2Fapi'
```

Comment:
The downloadUrl should be derived from the `about`, see the method in the MediaActionBar component.

Reply:
ok copied the method from MediaActionBar